src/extension.ts

```diff
@@ -318,6 +368,11 @@ function launchMetals(
   context.subscriptions.push(client.start());
 
   client.onReady().then(() => {
+    activation.status.dispose();
+    if (activation.shouldCreateNewProject)
+      client.sendRequest(ExecuteCommandRequest.type, {
```

Comment:
You can extract this to method as you duplicated this part also in `registerCommand`

Reply:
I am not sure it would help a lot here. It would be around the same amount of code.

Reply:
Aaaand I removed it altogether :sweat_smile: